File: apps/extension/samples/divide-by-zero.py
def divide_numbers(total: int, count: int) -> float:
    return total / count


def main() -> None:
    values = [12, 8, 4]
    print(f"First value: {values[0]}")
    print("About to divide by zero...")
    print(divide_numbers(sum(values), 0))


if __name__ == "__main__":
    main()
>

File: apps/extension/samples/DivideByZero.java
public class DivideByZero {
    static int divideNumbers(int total, int count) {
        return total / count;
    }

    public static void main(String[] args) {
        int[] values = {12, 8, 4};
        System.out.println("First value: " + values[0]);
        System.out.println("About to divide by zero...");
        System.out.println(divideNumbers(values[0] + values[1] + values[2], 0));
    }
}
